### PR TITLE
add notebook link

### PR DESCRIPTION
**Pull request by CalMacCQ**
Add notebook link under "more dcoumentation"

### CODE REVIEW

File: pytket/docs/index.rst [hunk 1]
```diff
@@ -128,6 +128,7 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     
     Manual <https://cqcl.github.io/pytket/manual/index.html>
     Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
+    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples>
```
Comment (cqc-melf) [inline pytket/docs/index.rst:131]: is there an index.html file somewhere in the directory with more information?

Comment (cqc-melf) [inline pytket/docs/index.rst:131]:
```suggestion
    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples#pytket-examples>
```

Comment (cqc-melf) [reply pytket/docs/index.rst:131]: How about this?

Comment (CalMacCQ) [reply pytket/docs/index.rst:131]: So I built the docs locally and then checked the built index.html file - the link seemed to work. Is this what you mean?

Comment (cqc-melf) [reply pytket/docs/index.rst:131]: I was thinking about changing the added link in a way that is point to an index.html file in the directory of the examples

Comment (cqc-melf) [reply pytket/docs/index.rst:131]: (see the comment below)

Comment (CalMacCQ) [reply pytket/docs/index.rst:131]: I tried out the suggested change and this seems to redirect to the README of the examples repo. Is this what you intended? If so this is fine with me.

Is there another file I should change as well?

**Comment by cqc-melf** [comment]
This is something we want in the release? Probably needs a rebase from develop and it should work then.

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**